apps/web/src/views/BuyCrypto/hooks/useProviderAvailability.ts

```diff
@@ -0,0 +1,39 @@
+// will cleanup urls later
+export async function fetchMoonpayAvailability(userIp: string): Promise<Response> {
+  // Fetch data from endpoint 1
+  const response = await fetch(`https://pcs-onramp-api.com/fetch-moonpay-availability?userIp=${userIp}`, {
```

Comment:
Make all the endpoint `https://pcs-onramp-api.com` in  [here](https://github.com/pancakeswap/pancake-frontend/blob/develop/apps/web/src/config/constants/endpoints.ts)